services/src/projects/project.rs

```diff
@@ -75,14 +75,14 @@ impl Project {
 
 #[derive(Debug, PartialEq, Serialize, Deserialize, Clone, ToSql, FromSql)]
 pub struct STRectangle {
-    pub spatial_reference: SpatialReference,
+    pub spatial_reference: SpatialReferenceOption,
     pub bounding_box: BoundingBox2D,
     pub time_interval: TimeInterval,
 }
 
 impl STRectangle {
     pub fn new<A, B>(
-        spatial_reference: SpatialReference,
+        spatial_reference: SpatialReferenceOption,
```

Comment:
You already say `time_start: A, ` … `where A: Into<TimeInstance>`,
you could be consistent and say `spatial_reference: S` … `,where S: Into<SpatialReferenceOption>`.
Then you can omit writing `wgs84().into()`.

Reply:
8966c1c9b1bff565fcdc74fd20f8d6b5cc41ddb6